translations/nl-be.json

```diff
@@ -245,6 +245,7 @@
   "no-case-type": "Geen dossiertype",
   "no-decision": "Nog geen beslissing voor dit agendapunt.",
   "no-documents-yet": "Er zijn nog geen documenten toegevoegd.",
+  "no-translation-activities-yet": "Er zijn nog geen vertalingsactiviteiten aangevraagd.",
```

Comment:
Deze vertaling, de gebruiker gaat misschien niet weten wat een "vertalingactiviteit" is.
`Er zijn nog geen vertalingen aangevraagd.` ?

Reply:
ligt meer in lijn met https://www.figma.com/file/D8NJqS92aZbzSx8GeSOaDr/%F0%9F%92%AF-Kaleidos-Designs-Digitale-Publicaties-v3?node-id=3565%3A73653

In de documenten tab moet de tekst staan: `Er zijn nog geen vertalingen.`
Daar staat nu "er zijn nog geen documenten"